src/main/java/racinggame/ds/Car.java

```diff
@@ -0,0 +1,56 @@
+package racinggame.ds;
+
+public class Car {
+  private String name;
+  private int position;
```

Comment:
```suggestion
  private final String name;
  private final int position;
```
객체의 상태를 final로 선언해주면 더 좋습니다

Reply:
position은 지속적으로 값이 변경되는 변수인데 final이 괜찮을까요??

Reply:
position은 변하기 때문에 final을 주지 않았고 이름은 final로 수정하였습니다.